tests/integration/nmstatectl_test.py

```diff
@@ -99,5 +114,12 @@ def test_show_command_only_non_existing():
     assert len(state[Constants.INTERFACES]) == 0
 
 
+def test_set_command_with_yaml_format():
+    ret = libcmd.exec_cmd(SET_CMD, stdin=ETH1_YAML_CONFIG)
+    rc, out, err = ret
```

Comment:
just a minor nitpick, IMHO it is not obvious what `err` means. It seems to be `stderr` but I would not assume this just based on the variable name. Using `stdout` and `stderr` here would be more obvious since the input to `exec_cmd` is also `stdin`.

Reply:
IMO `stderr` and `stdout` may be used as either reserved keywords or just as package names, it is "too" common to use it like this.
On the other hand, it is used as an argument name by other existing function (including `exec_cmd`).

I don't really know, it feel uncomfortable for me to use std* names, perhaps due to my c/c++ history. We will then argue if this is a stream or a string and never end this discussion.
Leaving the coder to choose a reasonable name is a good middle ground IMO.